### PR TITLE
node and group on/off/... methods return a status, added [] around va…

### PR DESCRIPTION
**Pull request by jimboca**
…l in error message so caller can tell if the device ends in space
Also commented out the program update that was causing for issue #7

### CODE REVIEW

**Comment by jimboca** [comment]
Sorry, I didn't notice there was a conflict.  Not sure why.

**Comment by rmkraus** [comment]
GitHub has [documented some instruction](https://help.github.com/articles/merging-an-upstream-repository-into-your-fork/) on how to resolve conflicts.

File: PyISY/Events/events.py [hunk 1]
```diff
@@ -78,9 +78,10 @@ def _routemsg(self, msg):
                 self.parent.variables._upmsg(xmldoc)
             elif '<id>' in msg:  # PROGRAM
                 self.parent.programs._upmsg(xmldoc)
-            else:  # SOMETHING HAPPENED WITH A PROGRAM FOLDER
+            #else:  # SOMETHING HAPPENED WITH A PROGRAM FOLDER
```
Comment (rmkraus) [inline PyISY/Events/events.py]: It is best to remove these lines rather than comment them out.

File: PyISY/Nodes/group.py [hunk 2]
```diff
@@ -83,8 +83,10 @@ def off(self):
         if response is None:
             self.parent.parent.log.warning('ISY could not turn off scene: ' +
                                            self._id)
+            return False
```
Comment (rmkraus) [inline PyISY/Nodes/group.py]: These are good. I'm not sure why I didn't have these already.

Comment (rmkraus) [inline PyISY/Nodes/group.py]: Should these also be added to the Programs?

**Comment by jimboca** [comment]
I see now my fork was not up to date since I did it yesterday, but I see there was another release.  Still getting familiar with git, hard to get old cvs/svn.

I will completely remove the lines, I just wasn't sure if you wanted them to remain in case the issue came back up.

Yes, Programs should also have it.  I'll be testing Alexa talking to programs tonight, so I'll need that.

**Comment by jimboca** [comment]
I think I have done all requested changes, although I can't test until tonight.

**Comment by jimboca** [comment]
I've run my program and everything seems to be working with these changes.

**Comment by rmkraus** [comment]
All looks good :+1: . Thanks.

**Comment by jimboca** [comment]
Awesome, thank you!

**Pull request merged**